src/main/java/Duke.java

```diff
@@ -1,10 +1,178 @@
+    private static final String ECHO_MESSAGE = " Echoing after you!\n";
+    private static final String QUIT_MESSAGE = " That was annoying huh...\n";
+    private static final String NO_TASK_MESSAGE = "No such task! You're not THAT productive...\n";
+    private static final String TASK_COMPLETE_MESSAGE = "Nice! You're a real champ for finishing this: \n";
+    private static final String BYE_MESSAGE = " Bye! Remember, stay out of fire, suuuuuuper high level " +
+            "tactic yea?\n";
+    private static final String ADD_MESSAGE = " Got it. I've added this task:\n   ";
+    private static final String INVALID_COMMAND = "I don't quite understand :/\n";
+    private static final String HELP_MESSAGE = " Here are the commands for the things I can do:\n" +
+            "    1. todo [task] = adds a Todo task\n" +
+            "    2. deadline [task] /[deadline] = adds a Deadline task\n" +
+            "    3. event [task] /[event range] = adds an Event task\n" +
+            "    4. done [taskIndex] = marks the inputted task as done\n" +
+            "    5. list = lists out all current tasks with their taskIndex\n" +
+            "    6. echo = turn into a huge cave and echo your inputs back to you!!\n" +
+            "    7. quit (only in echo mode) = turn back to normal and stop echoing\n" +
+            "    8. genshin = begin the genshin helper\n" +
+            "    9. bye = shuts me down... ;-;\n";
+
+
+    private static final Scanner sc = new Scanner(System.in);
+    
+
+
+    //Prints greeting message
+    public static void greeting() {
+        System.out.println(LINES + GREETING + LINES);
+    }
+
+    //Begins echo function
+    public static void echo() {
+        System.out.println(LINES + ECHO_MESSAGE + LINES);
+        boolean echoState = true;
+        while (echoState) {
+            String s = sc.nextLine();
+            if (s.equalsIgnoreCase("quit")) {
+                System.out.println(LINES + QUIT_MESSAGE + LINES);
+                echoState = false;
+            } else {
+                String output = LINES + " " + s + "\n" + LINES;
+                System.out.println(output);
+            }
+        }
+    }
+
+    //list function
+    public static void list(Task[] tasks) {
+        System.out.print(LINES);
+        for (int i = 0; tasks[i] != null; i++) {
+            int j = i + 1;
+            System.out.println(j + "." + tasks[i]);
+        }
+        System.out.print(LINES);
+    }
+
+    //Marks a specified task input as done
+    public static void markDone(String input, Task[] tasks, int taskCount) {
+        //isolate 'x' from 'done x', where x is a number
+        int index = Integer.parseInt(input.substring(5));
+        if (index > taskCount) {
+            System.out.println(LINES + NO_TASK_MESSAGE + LINES);
+        }
+        else {
+            tasks[index - 1].markAsDone();
+            System.out.print(LINES + TASK_COMPLETE_MESSAGE + tasks[index-1] + "\n" + LINES);
+        }
```

Comment:
if-else format should be in the same standardized format as line 49-55. same for line 82-97